cmd/mtaBuild_test.go

```diff
@@ -43,34 +45,56 @@ func TestMarBuild(t *testing.T) {
 		existingFiles["package.json"] = "{\"name\": \"myName\", \"version\": \"1.2.3\"}"
 		fileUtils := MtaTestFileUtilsMock{existingFiles: existingFiles}
 
-		err := runMtaBuild(options, &cpe, &e, &fileUtils, &httpClient)
+		utils := newNpmMockUtilsBundle()
+		utils.execRunner = e
+
+		npmExecutor := npmExecutorMock{utils: utils, options: npmExecuteOptions{
+			install:            false,
+			runScripts:         []string{},
+			runOptions:         []string{},
+			defaultNpmRegistry: options.DefaultNpmRegistry,
+			sapNpmRegistry:     options.SapNpmRegistry,
+		}}
+
+		err := runMtaBuild(options, &cpe, &e, &fileUtils, &httpClient, &npmExecutor)
 
 		assert.Nil(t, err)
 
-		if assert.Len(t, e.Calls, 4) { // the second (unchecked) entry is the mta call
-			assert.Equal(t, "npm", e.Calls[1].Exec)
-			assert.Equal(t, []string{"config", "set", "registry", "https://example.org/npm"}, e.Calls[1].Params)
+		if assert.Len(t, npmExecutor.utils.execRunner.Calls, 3) { // the second (unchecked) entry is the mta call
+			assert.Equal(t, "npm", npmExecutor.utils.execRunner.Calls[1].Exec)
+			assert.Equal(t, []string{"config", "set", "registry", "https://example.org/npm"}, npmExecutor.utils.execRunner.Calls[1].Params)
 		}
 	})
 
 	t.Run("Provide SAP npm registry", func(t *testing.T) {
 
 		e := mock.ExecMockRunner{}
-		e.StdoutReturn = map[string]string{"npm config get @sap:registry": "undefined"}
+		e.StdoutReturn = map[string]string{"npm config get registry": "undefined"}
```

Comment:
Was it intentional to remove the `@sap:` from the registry?

Reply:
No that was by accident.